### DISCUSSION TITLE
How do I know what version of Debian a Linux Mint download will use?

### DISCUSSION THREAD

**Discussion by hirehamir** [linuxmint/discussions#373] [Questions] [answered]
How do I know what version of Debian a Linux Mint download will use?

**Comment by rcalixte** [comment, marked as answer]
Assuming this is for LMDE, [the Wikipedia page for Linux Mint](https://en.wikipedia.org/wiki/Linux_Mint) has an easy table with this information, attached below with the current information.

![image](https://github.com/linuxmint/discussions/assets/10281587/50b95962-1113-4794-9ab0-319b7e9d1ca8)

**Reply by hirehamir** [reply]
Thank you!
